Simplified/NYPLBook.h

```diff
@@ -19,6 +14,7 @@ typedef NS_ENUM(NSInteger, NYPLBookAvailabilityStatus) {
 
 @property (nonatomic, readonly) NYPLBookAcquisition *acquisition;
 @property (nonatomic, readonly) NSString *authors;
+@property (nonatomic, readonly) NSArray *authorLinks;
```

Comment:
One of the things `NYPLBook` is meant to do is provide information in a very concrete form in terms of how it will be later used by the application. I think it might make sense to have a separate `NYPLBookAuthor` data structure here that very precisely contains what we need instead of including links that require knowledge of `NYPLOPDS*` classes to use.